src/backlight/labelizer/ternary/dynamic_neutral.py

```diff
@@ -46,3 +46,33 @@ def neutral_window(self) -> str:
     @property
     def neutral_hard_limit(self) -> str:
         return self._params["neutral_hard_limit"]
+
+
+class MarketCloseAwareDynamicNeutralLabelizer(DynamicNeutralLabelizer):
+    def _calculate_dynamic_neutral_range(self, diff_abs: pd.Series) -> pd.Series:
+
+        df = pd.DataFrame(diff_abs.values, index=diff_abs.index, columns=["res"])
+        df.loc[:, "est"] = df.index.tz_convert("America/New_York")
```

Comment:
ntp:could be renamed to something like `ny_time`?

EST(Eastern Standard Time ) is 5 hours  behind UTC
EDT(Eastern Daylight Time) is 4 hours  behind UTC
NY time changes between EST and EDT throughout the year.

Refs:
https://www.timeanddate.com/time/zones/edt
https://www.timeanddate.com/time/zones/est

Reply:
Thank you for reviewing! I've amended the variable name as such.
I didn't know the word, `EDT`, it was a good lesson for me.